.circleci/config.yml

```diff
@@ -14,7 +14,10 @@ jobs:
      steps:
        - run: rm -rf /home/circleci/project/.git # CircleCI git caching is likely broken
        - checkout
-       - run: ci/do_circle_ci.sh bazel.release
+       - run:
+           command:
+             ci/do_circle_ci.sh bazel.release
+           no_output_timeout: 60m
```

Comment:
Why is this needed here and below now? Shouldn't Bazel still be outputing status?

Reply:
@htuch any chance to let the G internal gtest folks know about this apparent perf degradation?

Reply:
Yes, but can we quantify somehow?